Wikipedia/Code/NotificationsCenterCellViewModel+SwipeActionExtensions.swift

```diff
@@ -0,0 +1,445 @@
+
+import Foundation
+
+extension NotificationsCenterCellViewModel {
+    
+    enum SwipeAction {
```

Comment:
I'd like to propose renaming `SwipeAction` (and these related types/occurrences/file name) to something that doesn't reference `Swipe`, like `NotificationSheetAction`, `SheetAction`, or `ActionSheetAction` or something. My reason being, `Swipe` is slightly confusing in this context, as it implies a cell's swipe actions (like [existing API](https://developer.apple.com/documentation/uikit/uiswipeactionsconfiguration)) instead of what they are in this case which is sheet actions.

Since it's a nested type of `NotificationCenterCellViewModel`, maybe the short `SheetAction` is most direct. What do you think?

Reply:
@staykids that sounds good, I like `SheetAction` since sheet is a generic enough term to cover this action sheet and our future half sheet modal.

I did a mindless find & replace, but I can trim it down more if you think it helps. Seems like some of those local `var sheetActions` could be simplified to just `var actions`.